sympy/physics/mechanics/jointsmethod.py

```diff
@@ -0,0 +1,132 @@
+from sympy.physics.mechanics import KanesMethod
+
+__all__ = ['JointsMethod']
+
+
+class JointsMethod(object):
+    """
+    Joints method object.
+
+    This object is used to form the equations of motion of a system created
+    using Joints.
+
+    Parameters
+    ----------
+    joints: List of Joints
+        List of all the joints created must be passed as argument.
+    root_body: Body
+        Root body w.r.t which equations of motion much be generated.
+        Is necessary to remove the  ambiguity since user may not
+        connect all bodies using joints and thus, there is not
+        way to find the root_body.
+
+    Examples
+    --------
```

Comment:
Please explain what this creates. It seems to create a double pendulum where the base joint is a pin joint and the second joint is a spherical one. It also give full inertial properties of each body too.